### DISCUSSION TITLE
How to set column width when calling `print()`?

### DISCUSSION THREAD

**Discussion by dimidloc** [opensource9ja/danfojs#243] [Q&A]
```
s = new dfd.Series([1, 3, 5, undefined, 6, 8])
s.print()
```
How to set column width when calling the `print()` method of a Series?

**Comment by risenW** [comment]
@dimidloc Do you mean the number of rows displayed by print?

**Reply by dimidloc** [reply]
I mean the width of a column. If a certain column has a long text, print will truncate it. Thanks

**Reply by risenW** [reply]
Ah, you can't do that for now, I just fixed this in the new version I'm working on. Once released, you'll be able to do this.
